c/crypt/crypt_master_ubi8.4.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : crypt
+# Version       : b2862e3d0a77
+# Source repo   : https://github.com/xordataexchange/crypt
+# Tested on     : RHEL ubi 8.4
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Prashant Khoje <prashant.khoje@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# -----------------------------------------------------------------------------
+
+export PACKAGE_NAME=crypt
+# The requested version v0.0.3-0.20170626215501-b2862e3d0a77 isn't released on github.
```

Comment:
have you tried using pseudo version https://jfrog.com/blog/go-big-with-pseudo-versions-and-gocenter/ ?